docs/index.md

```diff
@@ -96,26 +78,20 @@ There is an orange arrow on the Distance Axis indicating the distance to the tar
 
 ![Return Plot](/ping-viewer/images/viewer/return-plot-closeup.png)
 
-The Return Plot displays the return strength vs distance of the most recent [**Profile**](#distance-and-profile-measurements) sample. The plot displays the measurement of only a single dependent variable (return strength), and is simply mirrored for ease of viewing. Stronger returns appear as wider traces.
+The Return Plot displays the return strength vs distance of the most recent profile sample. The plot displays the measurement of only a single dependent variable (return strength), and is simply mirrored for ease of viewing. Stronger returns appear as wider traces.
 
 **Note** You may observe a very strong return at the top of the screen (at zero distance, essentially touching the device); this return is from the Ping device itself. When the Ping device emits the acoustic pulse, the device is still vibrating or 'ringing' like a bell when it begins measuring the return signal. This residual energy in the vibrations of the Ping device body is picked up as a return signal until it decays away.
 
 #### Waterfall
 
 ![Waterfall](/ping-viewer/images/viewer/waterfall-closeup.png)
 
-The Waterfall is a three dimensional plot that occupies the main portion of the application window. The Waterfall plots consecutive [**Profile**](#distance-and-profile-measurements) samples (distance running vertically and color indicating signal strength). The horizontal axis is time; new data is displayed on the right edge of the Waterfall as older data moves to the left.
+The Waterfall is a three dimensional plot that occupies the main portion of the application window. The Waterfall plots consecutive profile samples (distance running vertically and color indicating signal strength). The horizontal axis is time; new data is displayed on the right edge of the Waterfall as older data moves to the left.
 
 The color scheme (gradient) of the Waterfall can be [customized](display-settings.md).
 
 ## Application and Device Settings
 
 See [here](display-settings.md) for instructions to switch between meters and feet for the application display.
 
-To adjust the update rate and other Ping device settings, visit the [device configuration](device-settings-ping1d) page.
-
-## Interpreting Profile Data
```

Comment:
I believe that such section is not only for ping1d

Reply:
> To adjust the update rate and other Ping device settings, visit the relevant page in the 'Device Settings' section.

Reply:
It is beyond the scope of this documentation.